app/app.css

```diff
@@ -23,3 +23,23 @@
   content: "|";
   padding-right: 0.3em;
 }
+
+
+.totalmembers{ text-align:center;}
+.table > thead > tr> th{text-align: center;}
+.table > tbody > tr> td{text-align: center;}
+.btn.btn-default.btn-block{text-align: center;}
```

Comment:
I'm assuming that you are trying to change some bootstrap, this is not the way to go about it.